src/main/java/com/team2/practice1/Helper.java

```diff
@@ -0,0 +1,75 @@
+package com.team2.practice1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class Helper {
+	
+	private WebDriver driver;
+	private WebDriverWait wait;
+	   
+	   public void waitPageToPresent(WebDriver driver, final int size, final List<String> existedHandles) {
```

Comment:
Different indentation here. Please use auto-format function (Ctrl + Shift + F), or go to menu bar (Source > Format)